src/main/java/emis.command/Command.java

```diff
@@ -0,0 +1,7 @@
+package main.java.emis.command;
+import main.java.emis.*;
```

Comment:
Perhaps it will be better to specify the files that are imported instead of using the * to import everything.